compiler/noirc_frontend/src/hir_def/types.rs

```diff
@@ -1752,6 +1761,8 @@ impl From<&Type> for PrintableType {
             Type::MutableReference(typ) => {
                 PrintableType::MutableReference { typ: Box::new(typ.as_ref().into()) }
             }
+            // Is this actually unreachable?
```

Comment:
I think it would be unreachable because `Type::Code` can't pass monomorphization?